org.eclipse.lyo.oslc4j.codegenerator/src/org/eclipse/lyo/oslc4j/codegenerator/services/fileServices.mtl

```diff
@@ -0,0 +1,10 @@
+[comment encoding = UTF-8 /]
+[module fileServices('http://org.eclipse.lyo/oslc4j/adaptorInterface')/]
+
+[query public setTargetFolder(path : String): Boolean
```

Comment:
that's not really a query but a side-effect

Reply:
yes. I needed to save the targetFolder, so I don't have to pass it around all over the acceleo code.